.github/workflows/close-incident-discussions.yml

```diff
@@ -0,0 +1,30 @@
+name: Close incident discussion after time elapsed
+
+on:
+  workflow_dispatch:
+  schedule:
+    # daily at midnight
```

Comment:
UTC? Or does Actions have a default timezone?

Reply:
UTC, I can update the comment